src/main.rs

```diff
@@ -11,6 +11,15 @@ extern crate fnv;
 #[cfg(feature = "sysalloc")]
 extern crate alloc_system;
 
+#[macro_export]
```

Comment:
Accept, _if_ moved to `verify`.  Would like to keep the scope of the dirty macros as small as possible.